streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -766,14 +750,10 @@ public void shouldNotEnforceRebalanceWhenCurrentlyRebalancing() throws Interrupt
             mockTime
         );
 
-        final Consumer<byte[], byte[]> mockConsumer = EasyMock.createNiceMock(Consumer.class);
-        expect(mockConsumer.poll(anyObject())).andStubReturn(ConsumerRecords.empty());
```

Comment:
You cannot remove this stub without replacement. In this way a poll on the consumer will return `null`. Some tests are failing because of that. You haven't seen failing them because of the removal of `@Parametrized`.